app/views/invitations/_invitation.html.erb

```diff
@@ -12,7 +12,8 @@
   <dt class="heading"><%= t(".sent_to") %></dt>
   <dd>
     <% if @invitation.redeemed_at %>
-      <%= @invitation.invitee_email %>
+      <%= link_to @invitation.invitee_email,
+            find_admin_invitations_path(invitation: { invitee_email: @invitation.invitee_email }) %>
```

Comment:
If the invitee email is empty, e.g. because the users gave the invite link directly to whoever redeemed the invite, this will now display the raw link (`/admin/invitations/find?invitation%5Binvitee_email%5D=`). The field should instead be empty to match how it currently looks like on production:

<img width="430" height="496" alt="Screenshot_20251124_201301" src="https://github.com/user-attachments/assets/2c72e63a-deb5-4b47-9fb7-334a05986d40" />